apps/modernization-ui/src/design-system/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Selectable } from 'options';
+import styles from './checkbox.module.scss';
+import classNames from 'classnames';
+
+type Props = {
+    option: Selectable;
+    selected: boolean;
+    onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
suggestion:  It might be helpful to declare the onChange as `(value: Selectable | null) => void` so that a single checkbox could work with a `Controller` from react-form-hook`.  It may also be helpful to have an event for selection so that it can work seamlessly with `CheckboxGroup`.  

Something like 

```typescript
type onSelect = (type: 'select' | 'deselect', value: Selectable) => void;
```